charts/ocis/docs/values.adoc.yaml

```diff
@@ -419,6 +419,10 @@ features:
       # Set to "jwt" or "none"
       accessTokenVerifyMethod: "jwt"
 
+      #-- Do not look up user claims at the userinfo endpoint and directly read them from the access token.
```

Comment:
must be `# -- `, otherwise it will not be recognized as a doc string, see also the generated output in 'charts/ocis/docs`

Reply:
Thanks, fixed.